### PR TITLE
Vahe:  tasks 51-60

### PR DESCRIPTION
**Pull request by hsp-vahe**

### CODE REVIEW

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_53.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package main.java.am.hsp.apps.seua.tasks_51_60;
+
+import main.java.am.hsp.apps.seua.base.TaskDigit;
+
+/**
+ *
```
[the remark this answers is not in the document]
Comment (hsp-vahe) [reply src/main/java/am/hsp/apps/seua/tasks_51_60/Task_53.java]: Couldn`t anderstand what must we do?

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	}
+    
+	/*
+	 * (non-Javadoc)
+	 * @see main.java.am.hsp.apps.seua.base.TaskBase#solve()
+	 */
+	@Override
+	public void solve() {
+		int number = getUserInput("Please enter the number");
+		int min = getMin(getTens(number), getUnits(number), getHundreds(number));
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java]: getMinDigit

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		int mid = getMiddleDigit(number);
+		int max = getMax(getTens(number), getUnits(number), getHundreds(number));
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java]: getMaxDigit

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+		System.out.println(min  +  " " + mid + " " + max);
+		
+	}
+	
+	/**
+	 * Get the middle digit of the number.
+	 */
+	private int getMiddleDigit(int number) {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java]: No need for this at all

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    
+	/*
+	 * (non-Javadoc)
+	 * @see main.java.am.hsp.apps.seua.base.TaskBase#solve()
+	 */
+	@Override
+	public void solve() {
+		int number = getUserInput("Please enter the number");
+		int min = getMin(getTens(number), getUnits(number), getHundreds(number));
+		int mid = getMiddleDigit(number);
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java]: getSumOfDigits - min - max

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_60.java [hunk 6]
```diff
@@ -0,0 +1,39 @@
+package main.java.am.hsp.apps.seua.tasks_51_60;
+
+import main.java.am.hsp.apps.seua.base.TaskDigit;
+
+/**
+ * return the given number`s digits discendingly
+ * @author vahe
+ *
+ */
+public class Task_60 extends TaskDigit {
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_60.java]: same here

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    
+	/*
+	 * (non-Javadoc)
+	 * @see main.java.am.hsp.apps.seua.base.TaskBase#solve()
+	 */
+	@Override
+	public void solve() {
+		int number = getUserInput("Please enter the number");
+		int min = getMin(getTens(number), getUnits(number), getHundreds(number));
+		int max = getMax(getTens(number), getUnits(number), getHundreds(number));
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_59.java]: You are not fixing what we have discussed so many times. As I said, you are using the SAME method several times which causes our program to do the SAME calculations several times. Instead you need to store the data returned from the methods in to the variables and then use that variables in your further code...

File: src/main/java/am/hsp/apps/seua/tasks_51_60/Task_60.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+	/*
+	 * (non-Javadoc)
+	 * @see main.java.am.hsp.apps.seua.base.TaskBase#solve()
+	 */
+	@Override
+	public void solve() {
+		int number = getUserInput("Please enter the number");
+		int min = getMin(getTens(number), getUnits(number), getHundreds(number));
+		int max = getMax(getTens(number), getUnits(number), getHundreds(number));
```
Comment (hsp-harut) [inline src/main/java/am/hsp/apps/seua/tasks_51_60/Task_60.java]: same

**Pull request merged**